ietf/group/models.py

```diff
@@ -282,14 +283,14 @@ class GroupFeatures(models.Model):
     agenda_type             = models.ForeignKey(AgendaTypeName, null=True, default="ietf", on_delete=CASCADE)
     about_page              = models.CharField(max_length=64, blank=False, default="ietf.group.views.group_about" )
     default_tab             = models.CharField(max_length=64, blank=False, default="ietf.group.views.group_about" )
-    material_types          = jsonfield.JSONField(max_length=64, blank=False, default=["slides"])
-    default_used_roles      = jsonfield.JSONField(max_length=256, blank=False, default=[])
-    admin_roles             = jsonfield.JSONField(max_length=64, blank=False, default=["chair"]) # Trac Admin
-    docman_roles            = jsonfield.JSONField(max_length=128, blank=False, default=["ad","chair","delegate","secr"])
-    groupman_roles          = jsonfield.JSONField(max_length=128, blank=False, default=["ad","chair",])
-    groupman_authroles      = jsonfield.JSONField(max_length=128, blank=False, default=["Secretariat",])
-    matman_roles            = jsonfield.JSONField(max_length=128, blank=False, default=["ad","chair","delegate","secr"])
-    role_order              = jsonfield.JSONField(max_length=128, blank=False, default=["chair","secr","member"],
+    material_types          = IETFJSONField(max_length=64, allowed_empty_values=[[], {}], blank=False, default=["slides"])
+    default_used_roles      = IETFJSONField(max_length=256, allowed_empty_values=[[], {}], blank=False, default=[])
+    admin_roles             = IETFJSONField(max_length=64, allowed_empty_values=[[], {}], blank=False, default=["chair"]) # Trac Admin
+    docman_roles            = IETFJSONField(max_length=128, allowed_empty_values=[[], {}], blank=False, default=["ad","chair","delegate","secr"])
+    groupman_roles          = IETFJSONField(max_length=128, allowed_empty_values=[[], {}], blank=False, default=["ad","chair",])
+    groupman_authroles      = IETFJSONField(max_length=128, allowed_empty_values=[[], {}], blank=False, default=["Secretariat",])
+    matman_roles            = IETFJSONField(max_length=128, allowed_empty_values=[[], {}], blank=False, default=["ad","chair","delegate","secr"])
+    role_order              = IETFJSONField(max_length=128, allowed_empty_values=[[], {}], blank=False, default=["chair","secr","member"],
```

Comment:
Is anyone likely to want to use `IETFJSONField` *without* setting `allowed_empty_values`? Would it make sense to make that the default so the current uses don't have to repeat the list?

Reply:
My thinking is that if Robert agrees (which I expect), I'll submit changes upstream.  For the general case, the upstream will want to retain the existing behavior.  Assuming the upstream takes the change, then either this can revert to a JSONField, or IETFJSONField can become a pass-through for JSONField.